mne/viz/_figure.py

```diff
@@ -2097,6 +2093,41 @@ def _blit_vline(self, visible):
         self.mne.vline_visible = visible
 
 
+class MNEPSDFigure(MNEFigure):
+    """Interactive figure for power spectral density plots."""
+
+    def __init__(self, inst, n_axes, figsize, **kwargs):
+        super().__init__(figsize=figsize, inst=inst, **kwargs)
+
+        # AXES: default margins (inches)
+        l_margin = 0.8
+        r_margin = 0.2
+        b_margin = 0.65
+        t_margin = 0.35
+        # AXES: default margins (figure-relative coordinates)
+        left = self._inch_to_rel(l_margin)
+        right = 1 - self._inch_to_rel(r_margin)
+        bottom = self._inch_to_rel(b_margin, horiz=False)
+        top = 1 - self._inch_to_rel(t_margin, horiz=False)
+        # AXES: make subplots
+        axes = [self.add_subplot(n_axes, 1, 1)]
+        for ix in range(1, n_axes):
+            axes.append(self.add_subplot(n_axes, 1, ix + 1, sharex=axes[0]))
+        self.subplotpars.update(left=left, bottom=bottom, top=top, right=right,
+                                hspace=0.4)
+        # save useful things
+        vars(self.mne).update(ax_list=axes)
```

Comment:
Can you add a comment why you need to use `vars` rather than just doing `self.mne.ax_list = axes` or so?

Reply:
no good reason. I was mimicking `MNEBrowseFigure` (in which there were a lot of things to add at the end of `__init__`, and so using `vars` there made it much more concise).  Here doing a simple assignment would be fine.